src/main/java/com/zaxxer/nuprocess/linux/ProcessEpoll.java

```diff
@@ -62,7 +64,11 @@
       }
 
       triggeredEvent = new EpollEvent();
-      deadPool = new LinkedList<LinuxProcess>();
+      // disable auto-read and auto-write for triggeredEvent; process() will explicitly call read()
+      // only if epoll_wait returns 1, and only call write() before invoking epoll_ctl
+      triggeredEvent.setAutoSynch(false);
```

Comment:
This reduces `SharedMemory` allocations in `process`:
* The contents of the structure aren't read by `epoll_wait`, only written to, so they're irrelevant and a `write()` is nothing but overhead
* If `epoll_wait` returns <1, there's no point in reading the native memory back onto the heap
* Native memory is explicitly written prior to calling `epoll_ctl` to register a continued interest in `stdin` availability, but is not read back after the call